templates/yum_importer.json.erb

```diff
@@ -15,8 +15,13 @@
     "proxy_username": null,
     <% end %>
     <% unless [nil, :undefined, :undef, ''].include?(scope['pulp::proxy_password']) -%>
-    "proxy_password": "<%= scope['pulp::proxy_password'] %>"
+    "proxy_password": "<%= scope['pulp::proxy_password'] %>",
     <% else %>
-    "proxy_password": null
+    "proxy_password": null,
+    <% end %>
+    <% unless [nil, :undefined, :undef, ''].include?(scope[' $real_yum_max_speed']) -%>
```

Comment:
Use `scope['pulp::real_yum_max_speed'] `, it follows the pattern of all the other uses of `scope` in this file.    i.e., no $, and definitely no spaces.

The documentation is here https://docs.puppet.com/puppet/latest/lang_template_erb.html#scopevariable-or-scopelookupvarvariable